app/controllers/batch_processes_controller.rb

```diff
@@ -64,8 +69,12 @@ def set_batch_process
 
     def set_parent_object
       @parent_object = ParentObject.find(params[:oid])
```

Comment:
I think we still need this if we want to show historical batch process data for parent objects that have since been deleted